src/gpu/intel/jit/ir/message.hpp

```diff
@@ -163,8 +164,9 @@ class send_t : public func_impl_t {
     }
 
     static func_t make_2d(const hw_t &hw, send_op_t op, const type_t &type,
-            int surface_width, int surface_height, int surface_pitch, int width,
-            int height, int count, bool vnni, bool transpose, bool zero_out,
+            expr_t surface_width, expr_t surface_height, expr_t surface_pitch,
```

Comment:
```suggestion
            const expr_t &surface_width, const expr_t &surface_height, const expr_t &surface_pitch,
```

Reply:
I switched to using `std::move` in the assignment instead, as that pattern generally enables better C++ optimizations with r_values.